### PR TITLE
Get fileName from topology

### PR DESCRIPTION
**Pull request by hvangeffen**

### CODE REVIEW

File: src/lib/download/downloadFiles.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export async function downloadFileAttachment(
 
 export async function downloadFileWithXhr(
   url: string,
-  extension?: string,
+  fileName?: string,
```
Comment (ceesvoesenek) [inline src/lib/download/downloadFiles.ts:73]: It's not strictly the filename as it is appended to a timestamp? Maybe something like `filenameSuffix`?

File: src/stores/workflows.ts [hunk 2]
```diff
@@ -48,6 +48,7 @@ const useWorkflowsStore = defineStore('workflows', {
       type: WorkflowType,
       filter: PartialRunTaskFilter | PartialProcessDataFilter,
       body?: string,
```
Comment (ceesvoesenek) [inline src/stores/workflows.ts:50]: I'm not a big fan of having to pass `undefined` arguments to use a single optional argument later in the list. Maybe pass an options object with optional `body` and `fileName` (or `fileNameSuffix`..) fields?

**Review by ceesvoesenek — COMMENTED** [review]
I don't have so much context on what this is supposed to do, but I trust that that is fine.

Some small comments only

File: src/stores/workflows.ts (hunk 2 above)
Comment (hvangeffen) [reply src/stores/workflows.ts:50]: I agree, I wanted named params from Python at this moment

File: src/lib/download/downloadFiles.ts (hunk 1 above)
Comment (hvangeffen) [reply src/lib/download/downloadFiles.ts]: Changed the name

File: src/stores/workflows.ts (hunk 2 above)
Comment (hvangeffen) [reply src/stores/workflows.ts]: Added to an options object

**Review by ceesvoesenek — APPROVED** [review]

**Pull request merged**